### DISCUSSION TITLE
React v18 and v19 support?

### DISCUSSION THREAD

**Discussion by lanesawyer** [chanzuckerberg/sci-components#778] [General]
I saw the [React compiler](https://react.dev/learn/react-compiler) announcement and it looks amazing. Should make working with React a bit simpler!

I noticed `sci-components` is currently using React v17 and was wondering if there are plans for this library to support v18 and (ultimately) v19.

**Comment by masoudmanson** [comment]
I’m happy to share that I’ve already tested the SDS with a React 18 project, and it works perfectly! You should be able to use it without any issues in your React 18 applications.

As for React 19, we haven’t had a chance to test the SDS library with it yet. But don’t worry, we’ll definitely look into compatibility as soon as we can ✨🥸

Our goal is to keep SDS up-to-date and running smoothly with the latest versions of React.
